test/test.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+// const param = require('../greet.js');
+const hi = require('../lib/lib.js');
+const assert = require('assert');
+
+
+
+describe('module', function(){
+  describe('greet', function(){
+    it('should say Hello + name', function(){
+      assert.equal(hi.greet('Isak'), 'Hello ' + 'Isak');
```

Comment:
So nice test, note that you just want to test that your output it correct, so you don't have to concate 'Isak' to Hello. You can just test for 'Hello Isak'.